src/components/inputs/TextInput.js

```diff
@@ -163,6 +175,10 @@ export default class TextInput extends BaseInput {
     return typography.lineHeight;
   }
 
+  capitalize(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  }
+
```

Comment:
I think it's better to use
https://lodash.com/docs#capitalize

Reply:
Cool. Thanks!